<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-10-14T16:31:26Z" name="Default suite" finished-at="2024-10-14T16:32:06Z" duration-ms="40160">
    <groups>
    </groups>
    <test started-at="2024-10-14T16:31:26Z" name="Default test" finished-at="2024-10-14T16:32:06Z" duration-ms="40160">
      <class name="com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest@3943a2be]" started-at="2024-10-14T16:31:26Z" name="setup" finished-at="2024-10-14T16:31:52Z" duration-ms="25438" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="sumOfBooksWithAllCharges()[pri:0, instance:com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest@3943a2be]" started-at="2024-10-14T16:31:52Z" name="sumOfBooksWithAllCharges" finished-at="2024-10-14T16:31:52Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumOfBooksWithAllCharges -->
        <test-method signature="sumOfBooksWithoutAllCharges()[pri:0, instance:com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest@3943a2be]" started-at="2024-10-14T16:31:52Z" name="sumOfBooksWithoutAllCharges" finished-at="2024-10-14T16:32:05Z" duration-ms="13642" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- sumOfBooksWithoutAllCharges -->
        <test-method signature="totalSelectedBooksCount()[pri:0, instance:com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest@3943a2be]" started-at="2024-10-14T16:32:05Z" name="totalSelectedBooksCount" finished-at="2024-10-14T16:32:05Z" duration-ms="3" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Total Not Matched expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Total Not Matched expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest.totalSelectedBooksCount(Search_AddItemsInCartTest.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- totalSelectedBooksCount -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest@3943a2be]" started-at="2024-10-14T16:32:05Z" name="tearDown" finished-at="2024-10-14T16:32:06Z" duration-ms="1019" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.booksCart.TechVeritoAssignment.Search_AddItemsInCartTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
